contracts/token/psp22/src/extensions/flashmint.rs

```diff
@@ -0,0 +1,165 @@
+use crate::traits::*;
+use brush::traits::{
+    AccountId,
+    Balance,
+    Flush,
+};
+pub use common::errors::{
+    FlashBorrowerError,
+    FlashLenderError,
+    PSP22FlashmintError,
+};
+use ink_env::Error as EnvError;
+use ink_prelude::{
+    string::String,
+    vec::Vec,
+};
+
+#[brush::wrapper]
+pub type PSP22FlashMintCaller = dyn FlashLender + PSP22;
+
+#[brush::trait_definition]
+pub trait PSP22FlashMint: PSP22 + Flush + FlashLender {
```

Comment:
It should be `pub trait FlashLender: PSP22`. `PSP22` requires `Flush` by default=) You are implementing the `FlashLender` trait, so it should have the same name for correct selectors of methods. It is why it should be called `FlashLender`.

Reply:
ahaaaaaaaa